engine/src/main/java/org/terasology/logic/players/CameraClientSystem.java

```diff
@@ -65,15 +98,38 @@ public void resetCamera() {
     }
 
```

Comment:
@Cervator: A-ha! It registers it under the method name! Crucial information I was missing. Thank you.